packages/authentication/src/definitions.ts

```diff
@@ -106,6 +107,11 @@ export interface FirebaseAuthenticationPlugin {
    * @since 0.1.0
    */
   getIdToken(options?: GetIdTokenOptions): Promise<GetIdTokenResult>;
+  /**
+   * Fetches the Firebase Auth ID Token and other helper properties for getting different data associated with the token as well as all the decoded payload claims for the currently signed-in user.
+   * 
+   */
+  getIdTokenResult(options?: GetIdTokenOptions): Promise<GetIdTokenInfo>;
```

Comment:
```suggestion
  getIdTokenResult(options?: GetIdTokenResultOptions): Promise<GetIdTokenResultResult>;
```

Interfaces should always be named like the method with a postfix (`Options` /  `Result`).